### ISSUE TITLE
Chaos Space Marines, Daemons and Renegades & Heretics Progress list

### ISSUE THREAD

**Issue by NebSeele** [BSData/wh40k-7th-edition#462]
This thread is intended for three tasks.

<b><i>1.</i></b>To replace/merge issues #391 and #393 as the main issue tracker for the profile overhaul project, as well as to have a general place for any long going issues to be linked to for easier access.

This is the issue thread that would probably be left open. Even if the bugs get fixed, there's always better ways to improve the catalogues. Therefore, this thread is

<b><i>2.</i></b>intended to serve as a roadmap in that regard, which anyone willing to contribute or even post different ideas on how something may work can post.

In addition, the possibility of GW releasing new stuff for anyone (not just chaos), whether it's GW Main or Forgeworld, always means the possibility of new things to add. 

<b><i>3.</i></b> This thread is also to catalogue what needs to be done. I'll be trying my best to update the issue thread (as well as the catalogues obviously), so that any bugs, additions or enhancements are kept up to date.

Anyways (after my ramblings), Onto the Progress roadmap

(sidenote: atm this is what I had as a roadmap, if anyone wishes to add anything, feel free to)

<b>Codex: Chaos Space Marines</b>

<i>known Issues: </i> 
- [x] 1. Missing equipment profiles (mostly vehicle equipment links)
- [x] 2. Missing Frag Grenades profile. 
- [x] 3. Missing weapon profiles for Infernal Relic Predator guns and Dreadnought Close Combat Weapon on Forgeworld Dreadnoughts

<i>Ammendments to be made</i>
- [x] 1. Battle Formation character profiles not consistent with the newer styled profiles of the HQ and Special Character Models. These formations (and most likely the standard ones too) will need to be updated.
- [x] 2. Warhound Titan needs to be updated to Imperial Armour 13 ruleset.
- [x] 3. Forgeworld units NOT updated in Imperial Armour 13 need to be modified to match the new profile typeset.

<i>Additions to make</i>
- [x] 1. Include the Chaos Reaver Titan in the Lords of War entries
- [x] 2. Include the Legacies of Ruin from Imperial Armour 13
- [x] 3a. Include the Warzone: Pandorax Super-Heavies
- [x] 3b. Include the Warzone: Pandorax Battle Formations

<b>Supplements: Black Legion and Crimson Slaughter</b>

(note: Since both suffer the same core problem, I am grouping them till the profile overhaul project is complete)

<i>known Issues: </i> 
- [x] #393 Profiles missing / Profile overhaul project 

<b>Renegades and Heretics</b>

Since the Seige of Vraks books are no longer valid, the Renegades and Heretics list will need to be remade. The unit profiles and rules can be found in Imperial Armour 13.

<b>Codex: Chaos Daemons</b>

<i>known Issues: </i> 
No known issues at this time.

<i>Ammendments to be made</i>
- [x] 1. Forgeworld options for Blight Drones, Blood Slaughterers, Decimator Engine and Daemon Lords need to be updated to match the rules-set found in Imperial Armour 13
- [x] 2. (In matching with other repositories) Add the Hidden tags/show modifiers for Forgeworld specific units. Add flagging entry into the catalogue.

<i>Additions to make</i>
- [ ] 1. Add the Battle Formations from Apocalypse (2013)

<b><i>About the Profile Overhaul Project: </b></i>

This project is about the major overhaul of the catalogues, changing them from having single units with the possibility of duplicate blank profiles, if any profiles at all (which was the case at the time of the original write-up,) to having shared profiles and shared rules entries (as it seems to be the case here). Where something can potentially be a duplicate, it is then moved to the shared section and link. If it's unique to the unit, then it stays.

It is a largely necessary project in order to bring it into standard, to account for the compatibility of 7th Edition styled lists, any future potential detachments and formations, and finally,  in order to include the Apocalypse Battle Formations into the catalogues.

Core Rulebooks rules are still deliberately left blank (as it was intended originally with all the entries, as it was the standard at the time) and any rule with a table will often refer back to the originating book in the rules text.

This project applies primarily to the Chaos Space Marine Codex and Supplements, as renegades needs writing and daemons already had shared entries and profiles.

**Comment by amis92** [comment]
I do like what you wrote up here. However let me note: Issues aren't threads. They shouldn't be left open indefinitely. So the part of this being for everything connected to CSM/CD/R&H isn't valid.

I do know that we need a place to discuss things and have threads and so on, like a forum. Maybe we should adapt the Randomhit forum again as a place for more streamlined project overviews, plans etc. 

EDIT: I've also added checkboxes to your lists, so when you finish something, edit your comment and put 'x' between parentheses like so `[x]`

**Comment by ghost** [comment]
The Renegades and Heretics list is up now & I'd be grateful if someone with the book could check a couple of things.
It's not clear what FOC slot Spawn should be: they're in the Elite section, but the two rules for selecting them talk about non-compulsory Troops...
Secondly, you may purchase Fanatic or Feel No Pain for an Infantry Platoon, but should that be for the entire platoon, or for each squad?
The titans have to be updated, but otherwise the file should be complete.

Incremental modifiers for points (on things like Camo Nets for tank squadrons) look like they aren't working, but the correct value is added to your roster selection. It's the same issue as the ticket raised here: http://battlescribe.uservoice.com/forums/55066-general/suggestions/6449645-modified-point-cost-is-not-reflected-in-battlescri

**Comment by NebSeele** [comment]
Apologies for a late reply on my part, while I do check up for critical and quick fixes, the vast majority of my updates both catalogue wise and post can only be done on a weekend when I have the dedicated time and mindset to think about it without it affecting other things.  (That and I always seem to write text walls that I cannot tell how long until after the post, so yeah TL; DR warning)

Amis92:  I think we do need a talking point as you are right in that issues aren't thread. Still, this issue gives a general idea of what (in the immediate,) needs to be done as well as a roadmap on how it's coming along.

Khambatta: The spawn reads as though it should be Elite, only selectable once Master of Renegades are selected. It's only once the Tzeentch dedication is selected then it gets the 0-3 non-compulsory troops.

I cannot find the Infantry Platoon upgrade option for fanatic/FnP, however I would say it would be a squad thing.  
As for the titans, if I can find a summary or update on what's different about them from IA: Apoc and IA: 13, it shouldn't be an issue to update the renegades files.

Edit 2: I'll leave that for now actually. I need to get some of the CSM stuff done first. Thanks for writing the list up khambatta.

Sidenote: Great work HoboDaveGoBOOM. It's always good to have others help out, and you also caught things I hadn't noticed (and had to double check). I hope you stay around and help out more :)

**Comment by HoboDaveGoBOOM** [comment]
That's alright, was only small fixes. I'm happy to look over the renegades and heretics list. The Arch-demagogue devotions make things a pain. As I understand it the covenants can only be applied per squad. Also all options in a platoon are per unit as they reference "squad".
The "other" feel no pain option is only for select units if you have a heretek magus and is your primary detachment.
Confusing!

**Comment by ghost** [comment]
I've put the infantry options at squad level, rather than for the whole platoon, so no change needed there then :) The titans are the same as before, I've corrected some small errors & added a bit of info for daemons. I think it's done.
Looking over the Master of Renegades rule again, I'm convinced now that the Spawn are correct: regardless of whether the text is meant to say "Elites" or "Troops", the line about them being non-compulsory puts them into the 'No Force Org Slot'.

@NebSeele For Chaos Daemons or Chaos Space Marines updates: Blight Drones & Blood Slaughterers are unchanged from their data in the last IA Apocalypse book, except the requirement for Khorne Berserkers with the latter has been removed.

**Comment by NebSeele** [comment]
I've fixed up the blood slaughterers and blight drones in Codex CSM. With mainly the apoc options (and any missed bugs) left, I will take a break from the vanilla codex for a bit and begin fixing up the supplements.

**Comment by HoboDaveGoBOOM** [comment]
Something I have noticed, not entirely sure how to correct or if it can be corrected.
Cult troops that are unlocked by marks/characters are unlocked for the roster as opposed to the primary detachment (or force). So when using two CADs you get cult troop unlocks in both lists from one character.
Should this be added as a new issue, or is it beyond current limitations?

**Comment by ghost** [comment]
That is doable. You'd want to change the modifiers so they refer to _maxInForce_ and _Parent Force_\* instead of _maxInRoster_ and _Roster_. For what it's worth, I reckon this is the best new feature @Jonskichov has added.

*it's displayed as _force type_ on the modifier

**Comment by NebSeele** [comment]
I have changed the modifiers on the selections/hide tags from Roster to Force. It seems to work correctly when I select each detachment in increments (aka, add detachments to the roster when making it). 

There is a problem where if using to CAD's the unlocks occur in the second CAD regardless of which CAD had the modifiers though. (Side note: said bug here also occured when I was checking and fixing the catalogues for #491 so it could be a battlescribe bug).

**Comment by HoboDaveGoBOOM** [comment]
Noticing the same issue, which is why I was wondering if the fix described by @khambatta was the correct way to go about it. Should we roll it back? I'd like to think people use the rules and know what the can and can't add even if unlocked (That may however be wishful thinking).

**Comment by ghost** [comment]
You get the same weird behaviour either way. I just tested it out with Chosen Troops modifiers saying 'in force' and Plague Marine Troops saying 'in roster'. If you build a new roster with 2 CADs, then add Abaddon & Typhus to the 1st detachment, the unlocked options only seem to appear in the 2nd detachment...

Then I tried it with a Chaos Marine Apoc detachment & an Orks: the Plague Marines & Chosen both pop up on the Orks tree!

**Comment by NebSeele** [comment]
Since it affects the bottom force, the only workaround that I can think of is to have the list start with one force, then add multiple forces as needed (activating each flag as needed).

Since this affects the way the hide tags work on the CSM Elite to troop modifiers (you more or less NEED to know how you want your troop choices to be), I'm gonna add a failsafe/extended functionality in the show tab that allows for ALL those troop options to be seen (roster wise as well). 

Ultimately though, it's not the chaos files that are affected the hardest by this bug (Though the FW stuff does now bloat out the FA and HS slots a slight) but the Space Marines, who's entire layout and managability is sorta... reliant on the hide tags now.

**Comment by NebSeele** [comment]
<b>Update:</b>  Black Legions Profiles are now done barring Forgeworld, which Black Legion didn't really have any of (experimental Fire Raptor and Place-holder Dreadclaw).

Unless it's absolutely necessary to have [FW] in Black Legion, I am going to begin Profile overhauls on Crimson Slaughter before adding the Forgeworld options as core codex profiles are more important (that and barring artefacts, most of BL can be done on Vanilla for now).

**Comment by lacyj3** [comment]
Android platform. most recent build.

Plague Marines - Heavy weapon choices are not displaying the selected weapon profiles inside the box with the other weapons.  Example: Plasma Gun selected but weapon profile not shown.  It is showing a Combi-Bolter though its not selected.  May have the wrong weapon ID coded for the Plasma Gun showing it as Combi-Bolter.

Plague Marine Champion - Does not have a Plague Knife.

Plague Marine Heavy weapon wielders are losing their Plague Knife, blight grenades and krak grenades.

Obliterator - With the Mark of Nurgle is not showing/giving his +1 Toughness.  It also would be nice if it displayed his available weapon choices in his stat box as well.

Forgefiend in standard base configuration has TWO Hades Cannons.  

Chaos Landraider shows "Chaos Landraider: Capacity"  I imagine the intention was to show display capacity and restrictions.

Chaos Rhino - Does not list its transport capacity or list itself at a transport.

Feature request:  If you take The Gift of mutation put it on a line by itself and make room or leave space at he end of it to pencil in or write in what you rolled.  The suggestion carries over to the Rewards and Champion of Chaos boon rule as well.   The extra space is geared to those who print the army list.   For the ones who don't print.  Give an option to select the the Mutation, champion of chaos or rewards during army creation.  When you do the eye symbol to play your army it will show the mutation, boon or reward.

**Comment by senpai514** [comment]
If you find new bugs it is better to create new issues for them vice adding
them all onto another older thread for tracking purposes.

**Comment by Sheeshmode** [comment]
Renegades and heretics needs an update on it's point values. Currently a series of heavy weapons, special weapons and other upgrades are not priced correctly in the RaH list.

**Comment by ghost** [comment]
@Sheeshmode That's a bit vague.

**Comment by Sheeshmode** [comment]
Wait what?

**Comment by Sheeshmode** [comment]
Ah I see. Hold on.

**Comment by amis92** [comment]
@NebSeele I'll close the issue seeing as only one thing is left, and open up new one just for it.
